### DISCUSSION TITLE
【新人爆照】

### DISCUSSION THREAD

**Discussion by wangzihhy** [share121/inter-knot#2713] [其他]
![b13183917256b9dad41a6794b7dfc54](https://github.com/user-attachments/assets/19d42c48-4313-49c6-986d-a0e6d5d1b94c)

不小心打开摄像头了

**Comment by KELERSAMA** [comment]
目测学弟是新开学的楼主

**Comment by LilangAlan** [comment]
也没看到外道魔像呀（）

**Comment by seelesee** [comment]
没看见白绝啊？<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >
